src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -664,9 +664,11 @@ private function renderException(\Throwable $exception): void
     /**
      * Override this method if you want to define more error enhancers.
      *
+     * @internal
```

Comment:
Are there other places where an `@internal` function is used in another package?